gromacs/__init__.py

```diff
@@ -242,6 +242,28 @@ def stop_logging():
     log.clear_handlers(logger)  # this _should_ do the job...
 
 
+
+# Check if GMXRC was loaded and tries to load it otherwise. Runs GMXRC in a
+# subprocess and put environment variables loaded by it in this environment.
+import os, subprocess
+if 'GMXPREFIX' not in os.environ:
+    envvars = ['GMXPREFIX', 'GMXBIN', 'GMXLDLIB', 'GMXMAN', 'GMXDATA',
+               'GROMACS_DIR', 'LD_LIBRARY_PATH', 'MANPATH', 'PKG_CONFIG_PATH',
+               'GROMACS_DIR', 'PATH']
+    cmdargs = ['bash', '-c', ". /usr/local/gromacs/bin/GMXRC && echo %s" %
```

Comment:
Don't hard-code the path to GMXRC. In `bash` you can also source anything in your `PATH`, so try just
```python
cmdargs = ['bash', '-c', ". GMXRC && echo %s" %
```